xLights/controllers/ILightThat.cpp

```diff
@@ -0,0 +1,130 @@
+/***************************************************************
+ * This source files comes from the xLights project
+ * https://www.xlights.org
+ * https://github.com/xLightsSequencer/xLights
+ * See the github commit history for a record of contributing
+ * developers.
+ * Copyright claimed based on commit dates recorded in Github
+ * License: https://github.com/xLightsSequencer/xLights/blob/master/License.txt
+ **************************************************************/
+
+#include "ControllerCaps.h"
+#include "ILightThat.h"
+
+#include "../UtilFunctions.h"
+#include "../models/Model.h"
+#include "../models/ModelManager.h"
+#include "../outputs/ControllerEthernet.h"
+#include "../outputs/DDPOutput.h"
+#include "../outputs/Output.h"
+#include "../outputs/OutputManager.h"
+
+#include "../xSchedule/wxJSON/jsonreader.h"
+#include "../xSchedule/wxJSON/jsonwriter.h"
+
+#include <curl/curl.h>
+
+#include <log4cpp/Category.hh>
+#include <utils/CurlManager.h>
+
+#pragma region Constructors and Destructors
+ILightThat::ILightThat(const std::string& ip, const std::string& proxy) :
+    BaseController(ip, proxy)
+{
+    static log4cpp::Category& logger_base = log4cpp::Category::getInstance(std::string("log_base"));
+    std::string const json = GetURL("/settings");
+    if (!json.empty()) {
+        wxJSONValue jsonVal;
+        wxJSONReader reader;
+
+        reader.Parse(json, &jsonVal);
+        _connected = true;
+        
+        if (_connected) {
```

Comment:
.. you just set it to true the line above. No need for the if {}

Reply:
A fair point - Revised. 

Thanks :)